core/modules/wmgr/python/client.py

```diff
@@ -206,13 +207,6 @@ class ServerResponseError(ClientException):
     def __init__(self, msg, *args):
         ClientException.__init__(self, 'Server response error: ' + msg, *args)
 
-class DataFileError(ClientException):
-    """
-    Exception raised for errors in data file operations
-    """
-    def __init__(self, fileName, *args):
-        ClientException.__init__(self, 'Data file error: ' + fileName, *args)
-
```

Comment:
This exception was used for errors when opening file, now file is opened by client code, so errors in file opening will have to be handled by client.